### PR TITLE
Fix script typo on usb-drive functions: reset2defaults and usb_upgrade.

### PR DESCRIPTION
**Pull request by denniszollo**
The reset2defaults was only working when a firmware file existed on flash drive due to my typo.  This should fix.

### CODE REVIEW

**Comment by anonymous-travis-swiftnav** [comment]
## v1.1.23-2-g1390e5b
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v1.1.23-2-g1390e5b/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.1.23-2-g1390e5b/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.1.23-2-g1390e5b

File: board/piksiv3/rootfs-overlay/etc/init.d/reset2defaults.sh [hunk 1]
```diff
@@ -11,14 +11,14 @@ _dir_wait () {
...
-    until [ -x $FIRMWARE ] || [ $REPEATS -lt 1 ]; do
+    until [ -x $MNT_PATH ] || [ $REPEATS -lt 1 ]; do
        usleep $(( $LEN_TIMEOUT_MS * 1000 ))
        let REPEATS=$REPEATS-1
     done
 }
 
 # try and mount drive for up to 5 seconds or until success 
-_dir_wait 20 250 /media/sda1
+_dir_wait 20 250 $RESETFILE
```
Comment (anth-cole) [inline board/piksiv3/rootfs-overlay/etc/init.d/reset2defaults.sh]: so this will now wait 5 secs to see if the reset file shows up?

Comment (denniszollo) [inline board/piksiv3/rootfs-overlay/etc/init.d/reset2defaults.sh]: yes but it shouldn't block.

**Comment by denniszollo** [comment]
Finally was able to test successfully on the bench.  It works!

**Pull request merged**